File: Home Work/Assignment2_4.py
x=int(input("Enter the value of x: "))
n=int(input("Enter the value of n: "))
fact=1
c=1
sum=1
for i in range(1,n+1,1):
  fact=fact*i
  c=c*x
  sum=sum+(c/fact)
print(sum)

File: Home Work/CSE115/Assignment2_4.c
#include <stdio.h>
int main()
{
  int x,n,factorial=1,c=1;
  float sum=1.0;
  printf("Enter the value of x: ");
  scanf("%d",&x);
  printf("Enter the value of n: ");
  scanf("%d",&n);
  for(int i=1; i<=n; i++)
  {
    factorial*=i;
    c=c*x;
    sum=sum+(c*1.0/factorial);
  }
  printf("Sum is %.2f",sum);
  return 0;
}